src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -749,6 +753,8 @@ Construct an Archimedean solid with the name given by String `s` from the list
 below.  The polytopes are realized with floating point numbers and thus not
```

Comment:
```suggestion
below.  Some polytopes are realized with floating point numbers and thus not
```
Small correction (unrelated to the PR) but I just noticed because these are used for the (doc-)tests.

Maybe we can make proper ones like for the johnson solids for these (in another PR)?

The same applies to the catalan solids below.